checks/microbenchmarks/mpi/osu/osu_tests.py

```diff
@@ -72,7 +74,15 @@ def setup_compilers(self):
             environ_name = self.current_environ.name
 
             if environ_name.startswith('PrgEnv-'):
-                if environ_name != 'PrgEnv-nvhpc':
+                if environ_name.endswith('-ce'):
```

Comment:
I am not sure about this, but maybe it's better to have a feature for this env. How about `containerized_env` or `ce_env` or something like this?